tests/integ/test_tf.py

```diff
@@ -100,10 +101,10 @@ def test_mnist_with_checkpoint_config(
     training_job_name = unique_name_from_base("test-tf-sm-mnist")
     with tests.integ.timeout.timeout(minutes=tests.integ.TRAINING_DEFAULT_TIMEOUT_MINUTES):
         estimator.fit(inputs=inputs, job_name=training_job_name)
-    assert_s3_files_exist(
+    assert_s3_file_patterns_exist(
```

Comment:
This change was made because the indexing for checkpoint files and the actual files generated have been changed from TF 1 to TF 2.
This means that files such as `graph.pbtxt` and `model.ckpt-0.meta` are not created as part of the checkpoint.
This also means that the indices for checkpoints start from 1 instead of 0.